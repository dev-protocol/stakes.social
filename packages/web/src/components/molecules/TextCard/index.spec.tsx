import React from 'react'
import { render } from '@testing-library/react'
import { TextCard } from '.'
import 'src/__mocks__/window/matchMedia.mock'

describe(`${TextCard.name}`, () => {
  test('Snapshot', () => {
    const component = render(
      <TextCard name="npm" description="description" authentication="authentication" calculation={1} />
    )
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
