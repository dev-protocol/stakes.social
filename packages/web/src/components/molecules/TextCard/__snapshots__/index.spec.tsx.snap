// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextCard Snapshot 1`] = `
<body>
  <div>
    <div>
      <h2
        style="font-size: 36px; line-height: 48px; color: rgb(47, 128, 237);"
      >
        npm
      </h2>
      <div
        style="font-size: 18px; line-height: 24px; color: rgba(0, 0, 0, 0.85);"
      >
        description
      </div>
      <div
        style="font-size: 18px; line-height: 24px; color: rgba(0, 0, 0, 0.85);"
      >
        Authentication: 
        authentication
      </div>
      <div
        style="font-size: 18px; line-height: 24px; color: rgba(0, 0, 0, 0.85);"
      >
        Calculation: 
        1
      </div>
    </div>
  </div>
</body>
`;
